### DISCUSSION TITLE
S-132 Completed

### DISCUSSION THREAD

**Discussion by donfroula** [thehighnibble/thehighnibble.github.io#107] [Show and tell]
I finished the S-132! There was a crazy amount of soldering involved. Getting the ESP32 module soldered was challenging because you have to create a solder bridge on each pin across a gap to the PCB. Keep flux and solder braid handy!  

It is recommended to drill and tap a hole in the back plexiglass to mount it. Yikes! t seems to stay put OK friction fit to the connector.

Configuration and setup was very easy, following the instructions on the website. It worked perfectly the first time.

There are no instructions on what to do with the power jumper and USB keyboard reset jumper. I guess you install the jumper to power from the barrel connector?

I used heat sink compound behind the 7805 voltage regulator.

Regards.

Don
K9CLF

![IMG_0739](https://github.com/thehighnibble/thehighnibble.github.io/assets/9483397/f3cb6846-7cc0-40a1-846e-708559e4c42a)

![IMG_0740](https://github.com/thehighnibble/thehighnibble.github.io/assets/9483397/799cf284-3ad3-42b8-961f-d84dec8e5405)

![IMG_0741](https://github.com/thehighnibble/thehighnibble.github.io/assets/9483397/68925714-2589-41be-890b-2194955e89b5)

![IMG_0742](https://github.com/thehighnibble/thehighnibble.github.io/assets/9483397/865f4eca-301d-4817-966a-d9a51673f80b)

**Comment by dmcnaugh** [comment]
Glad you got it working first time.
Yes, JP1 is the +5VDC select that bypasses the 7805.
I see your point, this is documented for the VT132 but not the S-132, I will amend this. It should read:

- JP1 (5V Select) bypasses the LM7805 5VDC linear power regulator and should only be put in place if you are supplying regulated +5VDC to the S-132 via the DC Barrel Jack

The other jumper marked `KB Reset` is less critical or interesting.
As you have the USB-HID keyboard interface then you can fit it.
When fitted, it causes the the USB-HID interface to be reset when you reset the S-132 with the `Reset` button at the top edge of the PCB.
This can help with some fringe conditions that can occur where it appears the USB-HID interface has frozen after the S-132 is reset.

**Reply by donfroula** [reply]
Thank you for the clarification, Dave. A misunderstanding in how that jumper operates could be fatal. I imagine applying +12 volts to the barrel jack with jumper installed would fry the entire system pretty thoroughly. I decided to leave the barrel jack connection alone until receiving clarification, though looking at the traces makes it pretty clear that the jumper bypasses the 7805.

**Comment by nbreeden2** [comment]
Glad you got it done and jumpered. Craze amount of soldering is subjective. I soldered an entire Apple ][ rev 0 PCB in one pass.  :)
